packages/entity-browser/package.json

```diff
@@ -14,6 +14,7 @@
     "tocco-test-util": ">0.0.0",
     "tocco-theme": ">0.0.0",
     "tocco-ui": "0.1.22",
-    "tocco-util": "0.1.19"
+    "tocco-util": "0.1.19",
```

Comment:
i think we can introduce a "greater than" (>0.1.0). This way always the locale version is used